packages/app/src/specs/generators/component/ReactComponentGeneratorStepOne.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <div class="flex flex-col flex-grow justify-between">
+    <template v-if="generatedSpecError">
+      <EmptyGenerator
+        :gql="generateSpecFromSource.currentProject"
+        title=""
+        type="component"
+        :other-generators="false"
+        :spec-file-name="generatedSpecError.fileName"
+        @restart="cancelSpecNameCreation"
+        @updateTitle="(value) => emits('update:title', value)"
+      />
+    </template>
+
+    <template v-else>
+      <div class="flex-grow">
+        <div
+          v-if="generateSpecMutation.fetching.value"
+          class="mt-48px w-full inline-flex items-center justify-center"
+        >
+          <i-cy-loading_x16 class="h-48px mr-12px animate-spin w-48px" />
+          <p class="text-lg">
+            Loading
+          </p>
+        </div>
+        <ExpandableFileChooser
+          v-else-if="!result"
+          v-model:extensionPattern="extensionPattern"
+          :files="allFiles"
+          :loading="query.fetching.value"
+          @select-item="makeSpec"
+        />
+        <GeneratorSuccess
+          v-else
+          :file="result.file"
+        />
+      </div>
+      <div>
+        <StandardModalFooter
+          v-if="result"
+          class="flex gap-16px items-center"
+        >
+          <router-link
+            class="outline-none"
+            :to="{ name: 'SpecRunner', query: { file: result.file.relative?.replace(/\\/g, '/') }, params: { shouldShowTroubleRenderingAlert: true } }
```

Comment:
We have this regexp a lot, I wonder if we can take this chance to refactor it? It could be `toPosix` or `posixify` (sounds cooler).

![image](https://user-images.githubusercontent.com/19196536/205774845-677743bb-601b-475d-84c2-afc51401f8a0.png)